pkg/install/install.go

```diff
@@ -217,3 +177,30 @@ func runCommand(c *exec.Cmd) error {
 	}
 	return nil
 }
+
+// retryingApply retries a kubectl apply call with the given path 3 times, sleeping
+// for 10s between each try.
+func retryingApply(path string) error {
+	cmd := exec.Command("kubectl", "apply", "-f", path)
+	for i := 0; ; i++ {
+		err := runCommand(cmd)
+		if err == nil {
+			break
+		}
+		if i >= 3 {
+			return err
+		}
+		time.Sleep(10 * time.Second)
+	}
+	return nil
```

Comment:
can this be:

```suggestion
	var err error
	for i := 0; i <= 3 ; i++ {
		if err = runCommand(cmd); err == nil {
			break
		}
		
		time.Sleep(10 * time.Second)
	}
	return err
```

Reply:
Done (with a slight change). I personally prefer to have assignments to `err` outside of the `if v == x(); err == nil` construct, as that's typically used with a local-to-the-if block error variable.

Reply:
heh, I actually almost suggested it that way for the same reason - wfm